src/app/organisms/room/msgContent.ts

```diff
@@ -95,10 +96,11 @@ export const getVideoMsgContent = async (
       )
     );
     if (thumbContent && thumbContent.thumbnail_info) {
+      const dim = scaleDimension(videoEl.width, videoEl.height, 512, 32, 512, 32);
       thumbContent.thumbnail_info[MATRIX_BLUR_HASH_PROPERTY_NAME] = encodeBlurHash(
         videoEl,
-        512,
-        scaleYDimension(videoEl.videoWidth, 512, videoEl.videoHeight)
+        dim.w,
+        dim.h,
       );
```

Comment:
I haven't actually tried this out with any videos, but felt that it would make sense to use the same scaling as for images for consistency, since videos are really just moving images. I added a bit larger minimum size in the hope that video controls, at least the play/pause button, are available. Although this isn't tested and may differ from browser to browser anyway.